setup.cfg

```diff
@@ -42,7 +42,7 @@ install_requires =
     # Temporary using this version as it gives huge performance boost
     pyLIQTR @ git+https://github.com/isi-usc-edu/pyLIQTR@fa74a62a5ecca6ed97822e877574735b5e4cf93b
     # Temporary using this version to get around the limit of length of h chains
-    openfermion @ git+https://github.com/zapatacomputing/OpenFermion@5b4aa97eb568baee576b04553ddf3055216eca3e
+    openfermion==1.5.1
```

Comment:
Just leaving a note here for future reference: I confirmed that 1.5.0 works but it looks like we can't go lower than 1.5.0 because of a dependency conflict with orquestra-cirq:
```
    openfermion 1.4.0 depends on cirq-google<=0.14.0 and >=0.12.0
    orquestra-cirq 0.8.0 depends on cirq-google~=1.0.0
```